dracut-saltboot/saltboot/module-setup.sh

```diff
@@ -47,6 +47,6 @@ install() {
     inst_hook cmdline 91 "$moddir/saltboot-root.sh"
     inst_hook pre-mount 99 "$moddir/saltboot.sh"
 
-    echo "rd.neednet=1" > "${initdir}/etc/cmdline.d/50saltboot.conf"
+    echo "rd.neednet=1 rd.auto" > "${initdir}/etc/cmdline.d/50saltboot.conf"
```

Comment:
Can't this have some unintended side effects? There must be reason why it was disabled by default in dracut.

According to https://github.com/coreos/bugs/issues/2099#issuecomment-322636864 it is subject to races and recommends using udev rules for RAID configuration.
Following to https://github.com/coreos/bootengine/pull/130 . Can't we use something like that?

Reply:
Salt configuration is typically on root filesystem but there can be exceptions - for example if root is encrypted, so it is better to start everything.
If anything is assembled incorrectly, saltboot state can fix it.
In typical configuration with one disk the option has no effect, there is nothing to assemble.
So I think it is safe.

Reply:
Ok, you convinced me.